packages/x-components/src/store/store.types.ts

```diff
@@ -30,6 +31,28 @@ export interface RootXStoreState {
  */
 export type RootStoreStateAndGetters = Pick<Store<RootXStoreState>, 'state' | 'getters'>;
 
+/**
+ * State and Getters Store type for {@link RootXStoreState} and payload and metadata types.
```

Comment:
The link is wrong as it has to be `RootStoreStateAndGetters` but I'd put I'd write its purpose instead of what it has in the interface.